### ISSUE TITLE
OMG This is AMAZING. Game Changing. Is it possible to Beazer some parts of the line. 

### ISSUE THREAD

**Issue by xtianus79** [pixijs/graphics-smooth#37]
So first, off this plugin is game changing. To me this is something that could compete with D3 in a way, I think. 

Here are some pics of a quick script I did to test it out for my use case. It's just what I need. Except for 1 thing. I don't need to remind you what that looks like without this plugin. 

However, you'll notices that the lines that break in-between the amplitudes that "arc" are at an angle and it would be great if those could curve. will BiezerTo and QuadracticTo also work here? 

![download (1)](https://user-images.githubusercontent.com/8476336/217439526-59aca5e6-1a4b-49f0-93db-bc9540693e8b.png)
![download](https://user-images.githubusercontent.com/8476336/217439528-2153d662-d822-495a-8c9a-27f981717af3.png)

**Comment by ivanpopelyshev** [comment]
Thank you!

the lib was made for plots, and was made as improvement of https://github.com/gameofbombs/pixi-candles 

Unfortunately, i didnt have time to make better geometrization of curves, so you have to tune curves params for your case. As for doing curves on shader, well, i had that idea too but.. time. I'm super-busy, and I'm grateful that @dev7355608 takes care about the lib )

**Comment by ivanpopelyshev** [comment]
Yes, that's right, I have PTSD after trying to make plot-lib, API's of modern plot libs is that bad. Hope someone can take my results and continue it further :)
